index.html

```diff
@@ -1133,6 +1133,55 @@ <h3>Revocation Bitstring Length</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Unnecessary Correlation</h3>
+
+      <p>
+There are a number of global identifiers used in a status list entry, defined in
+Section [[[#bitstringstatuslistentry]]], that can be used across [=verifiers=]
+to correlate [=subjects=]. Some of these properties include `id`,
+`statusListIndex`, and `statusListCredential`.
+      </p>
+
+      <p>
+In some cases, when [=presenting=] a [=verifiable credential=] that contains
+other global identifiers (such as a driver's license identification number),
+using other global identifiers for the status list information does not cause
+further correlation harms since the only thing required for correlation is a
+single globally unique identifier.
+      </p>
+
+      <p>
+When these global identifiers are used in [=presentations=] that utilize
+[=selective disclosure=] or [=unlinkable disclosure=], they can violate privacy
+expectations. [=Issuers=] are urged to enable status information to be
+selectively disclosable when there is an expectation that a particular
+[=verifiable credential=] can be disclosed in a way that avoids
+correlation, such as proving that an individual is above a certain age.
+[=Verifiers=] can require that status information is revealed in situations that
+require them to know the current status of a credential, and the holder might
+then consent to revealing that information or might refuse to do so for a given
+transaction. In all cases, both [=issuers=] and [=verifiers=] are urged to
+ensure that the use of global identifiers is avoided when a particular exchange
+does not require correlation.
```

Comment:
```suggestion
expectations. [=Issuers=] are urged to enable status information to be
selectively disclosable/concealable when a particular [=verifiable credential=]
is expected to be disclosed in a way that doesn't need correlation, such as when
proving that an individual is above a certain age. [=Verifiers=] can require
that status information be revealed in situations that require them to know the
current status of a credential, and the holder might then consent or refuse to
reveal that information for a given transaction. In all cases, both [=issuers=]
and [=verifiers=] are urged to ensure that the use of global identifiers is
avoided when a particular exchange does not require correlation.
```

Reply:
Incorporating what I take as the the spirit of @David-Chadwick's latest tweaks of line 1164-1165 --
```suggestion
expectations. [=Issuers=] are urged to enable status information to be
selectively disclosable/concealable when a particular [=verifiable credential=]
is expected to be disclosed in a way that doesn't need correlation, such as when
proving that an individual is above a certain age. [=Verifiers=] can require
that status information be revealed in situations that require them to know the
current status of a credential, and the holder might then consent or refuse to
reveal that information for a given transaction. In all cases, both [=issuers=]
and [=verifiers=] are urged to avoid the use of global identifiers in order to
prevent correlation, unless it is required for or by a particular exchange.
```

Reply:
@TallTed . Better, thanks